scapy/arch/windows/__init__.py

```diff
@@ -49,7 +49,7 @@ def _exec_query_ps(cmd, fields):
     for line in ps.stdout:
         if not line.strip(): #skip empty lines
             continue
-        l.append(line.split(':', 1)[1].strip())
+        l.append(line.split(':', 1)[-1].strip())
```

Comment:
The 'line.split(':', 1)[1]' and 'line.split(':', 1)[-1]'  is no difference, and the latter can avoid mistakes.